### ISSUE TITLE
is it still in development?

### ISSUE THREAD

**Issue by jacksolutions** [OdooCommunityWidgets/theme_material_design#2]
or stable version?

**Comment by lukebranch** [comment]
@jacksolutions ,

This project is still in development. I'll push some PR's to this module in a month or two.

Currently it's on hold due to other projects.

**Comment by jacksolutions** [comment]
its very bad to hear that odoo 9 community and enterprise is going to be a different UI..
does this repo proceeding in that line? is it changing the UI like in enterprise?

**Comment by lukebranch** [comment]
@jacksolutions ,

No, this repo will stay open-source. I simply do not have time to work on it right now as I have a number of other projects that i'm working on.

**Comment by jacksolutions** [comment]
i am also developer, if you can guide me how to proceed futher, i can do further development.

let me know how can i contribute..

Thanks

**Comment by lukebranch** [comment]
@jacksolutions ,

Thanks for the offer. Basically I am looking to create a second branch for this that will use an external dependency for product_multi_image:

https://github.com/pedrobaeza/product-attribute/tree/8.0-product_images/product_multi_image

This dependency is still under active development, however as soon as I can find time I will begin work on basing my frontend view on this module.

product_multi_image allows for more flexibility on where the images are stored, including, on disk, in database and external URL. For those with larger catalogs the second two options (on disk and external URL) will be better as it will stop the database from swelling up to unmanageable size, and slowing down Odoo in the process.

If you want to contribute i'd recommend taking a look at product_multi_image to get an understanding of how it works, as I will be basing all future versions with new features on this, and only keeping the current version there for bugfixes, and to make sure it continues to be supported in the long-term for those that are currently using it in production.

If you have any ideas/suggestions feel free to comment back in this issue thread so we can consider collaborating on some features.

**Comment by jacksolutions** [comment]
but this is not the work related to material design? i am interested more towards material design theme on odoo. let me know if i am wrong?

**Comment by lukebranch** [comment]
@jacksolutions ,

Sorry I was looking at the wrong repo when I typed that. My apologies.

The material design theme for Odoo will be based on the following HTML-only repo to begin with:

https://github.com/FezVrasta/bootstrap-material-design

I plan to integrate his work into a functional Odoo theme first, and then work on adding more Material design changes and improvements.

Unfortunately there has not been much work done on this repo yet due to my own personal time-constraints, however as soon as I can find some time to get started i'll update the repo.

I'd be happy to consider accepting PR's to this repo in the meantime though if you would like to do any work on it.

**Comment by cliffkujala** [comment]
@lukebranch have you had time to work on this one any further?

**Comment by lukebranch** [comment]
@cliffkujala ,

Sorry for the late reply. I have not had time to work on any of these repo's for the past couple of years due to other commitments. It was my plan originally to create a boilerplate 8.0 theme (back when 8.0 was the latest release), however due to time constraints this did not materialise.

I do plan on working on some of the website-related widgets and theme boilerplates again in 12.0 when multi-website is finally released, as they will provide me with a boilerplate for future projects.

Unfortunately this will not be for at least another 6 months or so, and I don't plan to work on releasing anything pre-Odoo 12.0.
